src/commands/build/wranglerjs/output.rs

```diff
@@ -28,23 +29,44 @@ impl WranglerjsOutput {
         self.errors.join("\n")
     }
 
-    pub fn script_size(&self) -> String {
+    pub fn bundle_size(&self) -> String {
         let mut e = ZlibEncoder::new(Vec::new(), Compression::default());
+
+        //approximation of how bundles are gzipped
+
         e.write_all(&self.script.as_bytes())
-            .expect("could not write buffer");
+            .expect("could not write script buffer");
+
+        if let Some(wasm) = &self.wasm {
+            e.write_all(wasm.to_owned().as_bytes())
+                .expect("could not write wasm buffer");
+        }
+
         let compressed_bytes = e.finish();
+        let compressed_size = compressed_bytes.unwrap().len();
+
+        let mut bytes_left = (MAX_BUNDLE_SIZE) - (compressed_size as isize);
+        let mut oversized = false;
 
-        match NumberPrefix::decimal(compressed_bytes.unwrap().len() as f64) {
+        if bytes_left < 0 {
+            oversized = true;
+            bytes_left = -bytes_left;
```

Comment:
Is there a builtin Math.abs in Rust?

Reply:
https://doc.rust-lang.org/std/primitive.i32.html#method.abs